package.json

```diff
@@ -30,5 +33,8 @@
     "webbluetooth"
   ],
   "author": "Vernier Software & Technology",
-  "homepage": "https://github.com/VernierST/godirect-js"
+  "homepage": "https://github.com/VernierST/godirect-js",
+  "dependencies": {
+    "core-js": "^2.6.1"
+  }
```

Comment:
hey I was just reviewing this again today and retested -- we don't need to install core-js as a dependency, this is was to solve and intermediate problem before I specified node 8.0.0 as the target :)

Reply:
@spectranaut that's great! I just updated this PR to remove that dependency